plugins/BEdita/API/src/Controller/ResourcesController.php

```diff
@@ -96,6 +99,9 @@ public function relationships()
         $id = $this->request->param('id');
         $relationship = $this->request->param('relationship');
 
+        $allowAssoc = Inflector::underscore($this->name);
+        $this->set(['_allowedAssociations' => [ $allowAssoc => [$allowAssoc]]]);
```

Comment:
Sometimes we would probably want to hide some entity/table associations from controller response, we'll think about that later.
Reverse model associations committed.